restpack_serializer.gemspec

```
@@ -17,15 +17,15 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency 'activesupport', ['>= 4.0.3', '< 7.2']
-  gem.add_dependency 'activerecord', ['>= 4.0.3', '< 7.2']
+  gem.add_dependency 'activesupport', ['>= 4.0.3', '< 8.1']
+  gem.add_dependency 'activerecord', ['>= 4.0.3', '< 8.1']
   gem.add_dependency 'kaminari', ['>= 0.17.0', '< 2.0']
 
   gem.add_development_dependency 'restpack_gem', '~> 0.0.9'
   gem.add_development_dependency 'rake', '~> 13'
   gem.add_development_dependency 'guard-rspec', '~> 4.7'
-  gem.add_development_dependency 'factory_girl', '~> 4.7'
-  gem.add_development_dependency 'sqlite3', '~> 1.3'
+  gem.add_development_dependency 'sqlite3', '~> 2.1'
+  gem.add_development_dependency 'factory_bot_rails', '~> 5.0'
```

```suggestion
  gem.add_development_dependency 'factory_bot', '~> 5.0'
```

There's no Rails app defined in this gem's test suite, so `factory_bot_rails` won't have any effect as all it does is [load a railtie](https://github.com/thoughtbot/factory_bot_rails/blob/v5.2.0/lib/factory_bot_rails.rb#L1).